### ISSUE TITLE
Docs: <table> of HandBook>Eveyday Types is outside the parent element's area

### ISSUE THREAD

**Issue by nerdchanii** [microsoft/TypeScript-Website#2409]
**Page URL:** 

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#interfaces

**Issue:** 

It is occurs when the browser is less than approximately 1390px.
The table does not scroll to the x-axis, and the user cannot see the overflow area.

**Recommended Fix:** 
I think we can fix it by adding the css code below.
```css 
.markdown table.full-width-table {
  ...
  table-layout: fixed;
}
```

Please refer to the additional captured image.
**Now**
<img width="1590" alt="image" src="https://user-images.githubusercontent.com/63002393/171975199-bfd9c4f2-2046-416c-b876-0605db613d2c.png">

**If you add the above code**
<img width="1587" alt="image" src="https://user-images.githubusercontent.com/63002393/171975231-0c4f2605-df80-42ae-b7ff-277aca6ada70.png">

P.S. There may be awkwardness in expression because I am not good at English. Please excuse me, thanks..!

**Comment by nerdchanii** [comment]
Ah... Do I chose the wrong type of issue?..😳

**Comment by orta** [comment]
No, this is a great issue report - I think you might be right with the CSS change, want to make a PR with it?

**Comment by nerdchanii** [comment]
@orta  Thank you for saying that and answering. I opened the PR. Please consider it. :)

**Comment by prathu9** [comment]
@nerdchanii  Hello, it is still not fixed

**Comment by nerdchanii** [comment]
> @nerdchanii  Hello, it is still not fixed

@prathu9 I solved this in #2410. It was approved but seems to have to wait for the test.

**Comment by shdq** [comment]
Hello there!

There is a problem with this whole `.full-width-table` class. Because of `display: table;` rule in `.full-width-table` class, `<table>` rules are not applied, and it turns off horizontal scrolling for tables on mobile devices and zoomed pages.  Right now, it is **completely impossible to reach the content of the table**. 

Screenshot:
<img width="822" alt="Screenshot 2022-10-23 at 16 28 33" src="https://user-images.githubusercontent.com/1219618/197392156-8d1bdef1-1cec-4d74-938c-f27d4e186219.png">

It seems like this PR doesn't completely solve the problem. It won't look good on mobiles from the UI perspective:
<img width="340" alt="Screenshot 2022-10-18 at 14 08 45" src="https://user-images.githubusercontent.com/1219618/197391126-d75bb28e-efe2-47ae-935e-e3f6a44e13c5.png">

A possible solution is to remove the `.full-width-table` class completely. For desktops, `display: table;` makes it full width, which improve visible quality of the content insignificantly.

Screenshot with `.full-width-table` class:
<img width="903" alt="Screenshot 2022-10-23 at 16 15 51" src="https://user-images.githubusercontent.com/1219618/197391565-b3d939ba-8250-4c90-9936-6227fed4c528.png">
Screenshot without `.full-width-table` class:
<img width="904" alt="Screenshot 2022-10-23 at 16 16 25" src="https://user-images.githubusercontent.com/1219618/197391566-57ea36c1-56b2-4bf0-920a-1d55bbb377cd.png">

It seems like styles for `<table>` element are more than enough. 

It should be fixed as soon as possible, because users can't see the content of table(s) on small screens.

Best, Sergei.
